bridge/deploy/bridge.yaml

```diff
@@ -46,6 +46,10 @@ spec:
            value: "true"
          - name: SHOW_API_TOKEN
            value: "true"
+         - name: KEPTN_INSTALLATION_TYPE
+           value: "QUALITY_GATES,CONTINUOUS_OPERATIONS,CONTINUOUS_DELIVERY"
```

Comment:
Same as in the other PR, there is a mismatch between specifying `QUALITY_GATES` here, but expecting it to be `QUALITY_GATES_ONLY` in the checks above.

Reply:
I fixed that! Thanks.